src/app/app.component.html

```diff
@@ -41,13 +41,15 @@
     <!-- Main content: shows in split-pane when sidebar route active -->
     <ion-split-pane when="lg" contentId="main" [disabled]="!sidebarRouter.isActivated">
       <div style="display: flex; flex-direction: column; height: 100%; width: 100%" id="main">
-        <plh-main-header></plh-main-header>
+        @if (headerConfig().show) {
+          <plh-main-header></plh-main-header>
```

Comment:
nit(non-blocking): given that the header might appear/disappear on a per-template basis it might be easier to use a `style.display` or `style.visibility` attribute to handle show/hide (assuming the header host is rendered as block or flex)